### PR TITLE
added eslintrc

### PR DESCRIPTION
**Pull request by vsnehil92**

### CODE REVIEW

File: eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "rallycoding"
```
Comment (ufocoder) [inline eslintrc]: `react-app` may be?

**Comment by ufocoder** [comment]
Please add lint command for Travis CI also

**Comment by nghduc97** [comment]
I think @ducthienbui97 should choose which eslint config to go with.

File: eslintrc (hunk 1 above)
Comment (vsnehil92) [reply eslintrc]: It can be any dictionary to varify I have been using  rallycoding for my projects

**Comment by vsnehil92** [comment]
@ufocoder can you help me with how to add lint command for Travis CI since I am new to Travis CI

**Comment by ufocoder** [comment]
Add lint `npm` command for lint. Then add `npm run lint` into script section in `.travis.yml`

```
script:
  - npm run lint
  .. 
```

**Comment by ducthienbui97** [comment]
this project's code is not compatible with your current choice of eslint. I suggest you take a look at [this link](https://docs.codeclimate.com/docs/eslint#section-adding-a-eslintrc-js) as we made it followed codeclimate default recommendations.

**Comment by vsnehil92** [comment]
"npm run lint" runs infinite times i can't understand why?

File: .travis.yml [hunk 2]
```diff
@@ -5,11 +5,12 @@ node_js:
 install:
   - yarn
 script:
+  - npm run lint
```
Comment (ducthienbui97) [inline .travis.yml]: I use yarn on travis, can you replace npm by yarn?

File: package.json [hunk 3]
```diff
@@ -14,11 +16,15 @@
     "react-scripts": "1.0.12"
   },
   "scripts": {
+    "lint": "npm run lint",
```
Comment (ducthienbui97) [inline package.json]: you are recursively call lint, it'll be infinite recursion.

**Review by ducthienbui97 — CHANGES_REQUESTED** [review]

**Comment by vsnehil92** [comment]
the lint is working in the code but the code does not follow the standards of js

**Comment by ducthienbui97** [comment]
@vsnehil92  yes, like I said, I used codeclimate's suggestions, so you should [check it out](https://docs.codeclimate.com/docs/eslint#section-adding-a-eslintrc-js).

**Pull request closed without merging**